armi/reactor/tests/test_components.py

```diff
@@ -1262,6 +1400,13 @@ def test_thermallyExpands(self):
 
 
 class TestRadialSegment(TestShapedComponent):
+    """Test radial segment.
+
+    .. test:: Compute properties of radial segment
+        :id: T_ARMI_COMP_FLUID0
+        :tests: R_ARMI_COMP_FLUID
+    """
+
```

Comment:
Delete this one.  As stated in your `impl` crumb, this just isn't the right place.

Thanks!